docs/synthetics/http-monitor.md

```diff
@@ -64,7 +64,7 @@ Conditions can be configured on the response fields and metrics. The conditions
 * **Metric** - Used to make sure the metrics are within the expected range.
 * **Response Header** - Can be used to compare the value of a specific header field against the expected value.
 * **Response Body** - Compare the response body against the specified value using an operator.
-* **Response Body JSON** - If the response body is JSON, [JSONPath](https://github.com/json-path/JsonPath) can be used to extract a specific value and compare against the expected value.
+* **Response Body JSON** - If the response body is JSON, [JSONPath](https://github.com/json-path/JsonPath) can be used to extract a specific value and compare against the expected value. More information on how to set these conditions up is provided in the [next subsection](#configuring-custom-alerts-based-on-json-response).
```

Comment:
This would link to a section on this page, just a few lines below. This also has nothing to do with custom metrics, but for creating monitor alert conditions based on the response.